ClinGen/clincoded
# Connects to #644. Connects to #1352. Connects to #1362. Updates to Power section of Case-Control evidence plus small changes to Group and Individual evidence.

Steps to test #644:
1. Select a gene-disease record in the GCI.
2. Click the "Group" header to add group evidence.
3. Scroll to the bottom of the page to confirm that the note in the "Additional Information" section is in a blue box.

Steps to test #1352:
1. Select a gene-disease record in the GCI.
2. Click the "Individual" header to add individual evidence.
3. Select "Yes" for the "Is this Individual a proband" field.
4. Add a variant, like ClinVar ID 139214, in the "Associated Variant(s)" section.
5. In the last field that appears, confirm that "de novo" is present and italicized in "If the answer to the above de novo question is yes,...".

Steps to test #1362:
1. Select a gene-disease record in the GCI.
2. Click the "Case-Control" header to add case-control evidence.
3. In the "Power" section, try various values in the required fields to confirm that the allele frequency value updates as the fields change. To test that the fields are required, try saving the evidence with invalid data (or none at all) in one or more of the fields.
